src/components/ImageUpload/ImageUpload.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+extends ../Base/InputBase.pug
+block input
+  .image-upload__wrapper
+  input.vfield__control.mb-3(
+    type="file",
+    :id="`${name}-image-upload`"
+    @change="onChangeImage"
+  )
+  .img-peview__wrapper(v-if="localValue && localValue.url")
```

Comment:
This component should be a very tricky one, where I think that the localValue should be the single URL like `https://picsum.photos/34/34` rather than an object